app.js

```diff
@@ -17,11 +17,11 @@
 
 "use strict";
 
-const getStdin = require("get-stdin");
+const fs = require("fs");
```

Comment:
Since this entry point is for a command line app, I'm fine with using `fs` and `readFileSync` on `0` - aka `stdin`